app/api/attendees.py

```diff
@@ -101,7 +102,13 @@ def query(self, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        query_ = self.session.query(TicketHolder)
+
+        if "Authorization" in request.headers and not (has_access('is_admin') or has_access('is_super_admin')):
```

Comment:
Looks good, but `"Authorization" in request.headers` is not needed elsewhere where same stuff is checked